lib/sw_indexer_engine.rb

```diff
@@ -32,4 +35,9 @@ def index(druid, targets = nil)
   def purl_model(druid)
     DiscoveryIndexer::InputXml::Purlxml.new(druid).load
   end
+
+  def update_marc_record(druid)
```

Comment:
Ugh. I assumed `#purl_model` was one of ours. Does it make sense to wrap that in a local class?